src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -260,7 +260,8 @@ int main(int argc, char **argv)
     if (iNumThreads > 0) {
```

Comment:
Would not be better to have the value `iNumThreads` updated once with a IF, and then call once `omp_set_num_threads(iNumThreads);`